terraform/environments/core-network-services/firewall.tf

```diff
@@ -299,6 +299,122 @@ resource "aws_networkfirewall_rule_group" "stateless_rules" {
             }
           }
         }
+        stateless_rule {
+          priority = 100
```

Comment:
This probably needs changing as it will conflict with an earlier rule that's got the same priority